service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapper.java

```diff
@@ -294,7 +296,7 @@ private CompoundStatementClassCode prepareClassCode(List<Observation> derivedObs
         }
 
         private CommentType prepareCommentType(Observation observation) {
-            if (observation.getComment().equals("EMPTY REPORT")) {
+            if (observation.getId().equals(DUMMY_OBSERVATION_ID)) {
```

Comment:
It's safer to check the ID here (than check the comment which is user entered/free text)